Source/Core/Core/IOS/ES/Formats.cpp

```diff
@@ -307,8 +307,8 @@ std::vector<u8> TicketReader::GetRawTicketView(u32 ticket_num) const
 std::string TicketReader::GetIssuer() const
 {
   const char* bytes =
-      reinterpret_cast<const char*>(m_bytes.data() + offsetof(Ticket, signature_issuer));
-  return std::string(bytes, strnlen(bytes, sizeof(Ticket::signature_issuer)));
+      reinterpret_cast<const char*>(m_bytes.data() + offsetof(Ticket, signature.issuer));
+  return std::string(bytes, strnlen(bytes, sizeof(Ticket::signature.issuer)));
```

Comment:
Out of curiosity, does `offsetof(field.sub_field)` do the right thing when `field` is _not_ the first member of that struct (imagine an additional `int` or whatever in front of `signature` inside `TMDHeader`/`Ticket`)? Or does it simply return `offsetof(sub_field)` for the given `typeof(field)`, which happens to be correct because `Ticket::signature` is the first member (so starting-offset 0 for either)?